rllib/env/single_agent_env_runner.py

```diff
@@ -127,32 +127,37 @@ def sample(
         *,
         num_timesteps: int = None,
         num_episodes: int = None,
-        explore: bool = True,
+        explore: bool = None,
```

Comment:
This will have an influence on the sampling of PPO and SAC. Have we made the changes there, too?

Reply:
It shouldn't. The default of the `AlgorithmConfig.explore` is always True, also for DQN/SAC. So if we change this to None, the EnvRunners will use that `AlgorithmConfig.explore` setting instead. For evaluation workers, normally, users override `explore=False`, so this should also be ok (eval workers will thus call the `forward_inference` method).